File: Course_01/Java Language/Estrutura sequencial/consumo.java
package consumo;

import java.util.Locale;
import java.util.Scanner;

public class Main {

	public static void main(String[] args) {
		
		Locale.setDefault(Locale.US);
		
		Scanner sc = new Scanner(System.in);
		
		int distancia;
		double combustivel, medio;
		
		System.out.print("Distancia percorrida: ");
		distancia = sc.nextInt();
		System.out.print("Combustivel gasto: ");
		combustivel = sc.nextDouble();
		
		medio = distancia / combustivel;
		
		System.out.println("Consumo medio = " + String.format("%.3f", medio));
		
		sc.close();
	}

}

File: Course_01/C Language/EstruturaSequencial/consumo.c
#include <stdio.h>
#include <stdlib.h>

int main()
{
    double dist, comb, consumo;

    printf("Distancia percorrida: ");
    scanf("%lf", &dist);
    printf("Combustivel gasto: ");
    scanf("%lf", &comb);

    consumo = dist / comb;

    printf("Consumo medio = %.3lf", consumo);

    return 0;
}
